packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2886,6 +2886,8 @@ class InputDecoration {
 
   /// Text that suggests what sort of input the field accepts.
   ///
+  /// Using [hintText] can help password manager like Samsung Pass to well detect and autofill the input.
```

Comment:
Maybe this should be included as part of the overall class documentation, rather than specifically for the `hintText` field (since someone struggling with this problem probably wouldn't know which class field to check for an answer).

Reply:
I need to move that inside TextField & TextFormField class ?

Reply:
I think adding it to the InputDecorator docs would be ideal.

https://github.com/flutter/flutter/blob/0baf7a59041478e0d2958106b957e70437cb28f5/packages/flutter/lib/src/material/input_decorator.dart#L1708-L1730

Reply:
Done :)